modules/siddhi-core/pom.xml

```diff
@@ -127,6 +128,29 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
+                <executions>
```

Comment:
We have similar configurations in the parent pom, 
https://github.com/siddhi-io/siddhi/blob/master/pom.xml#L265